test/sql/attach/attach_export_import.test

```diff
@@ -48,6 +48,9 @@ SELECT * FROM integers
 ----
 does not exist
 
+statement ok
+drop view integers_view;
```

Comment:
@Mytherin this is odd right?
The database we're importing has `integers_view`, and we already have `integers_view` defined still (without this change), so I would expect the `IMPORT DATABASE` call to fail ?

Reply:
That does sound familiar, I think we indeed have an open issue for that, thanks 👍